### ISSUE TITLE
Possible static attribute issue

### ISSUE THREAD

**Issue by danniel** [code4romania/catpol-declaratii#180]
In `models.py`:

```python
class Politician(models.Model):
    all_positions = []

    ....

    def add_position(self, position):
        self.all_positions.append(position)
```

I think that the `all_positions` list as currently defined will be static. So every time you append to it, the changes will be available for all Politician instances. Is this side-effect desirable for the current use case?

**Comment by danniel** [comment]
```python
>>> class Something:
...     a = []
...     def append(self, item):
...             self.a.append(item)
... 
>>> one = Something()
>>> one.append(12)
>>> one.append(13)
>>> one.a
[12, 13]
>>> two = Something()
>>> two.append(14)
>>> two.a
[12, 13, 14]
>>> 
```
This happens because `one.a` and `two.a` point to the same list.

**Comment by catileptic** [comment]
You're right, this was my mistake. 

Adding `all_positions` as an instance variable will solve this. That means it should be initialized in the `__init__` (or via some other method that would make it an instance variable).

**Comment by catileptic** [comment]
Closed by #181
